spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -16,8 +16,9 @@
 
 package org.springframework.cloud.gcp.storage;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import java.util.logging.Logger;
```

Comment:
That is preferred from end-user perspective, for their Spring Boot applications.
We develop here *library*, and it would be better to use as light API for logging as possible to simplify bridging to the target user preferences for logging system: https://jira.spring.io/browse/SPR-14512